examples/AspNetCore/Controllers/WeatherForecastController.cs

```diff
@@ -16,6 +16,9 @@
 
 namespace Examples.AspNetCore.Controllers;
 
+using System.Diagnostics;
+using System.Diagnostics.Metrics;
+using Examples.AspNetCore;
```

Comment:
I guess the intention is:

```suggestion
namespace Examples.AspNetCore;
```